app/views/layouts/_social_media.html.erb

```diff
@@ -1,6 +1,6 @@
 <% compact_margin = Settings::General.social_media_handles.values.compact_blank.count > 5 %>
 <% Settings::General.social_media_handles.each do |social_media_type, handle| %>
-  <% next if handle.blank? %>
+  <% #next if handle.blank? %>
```

Comment:
Was this unintentional?

Reply:
Good call, this was only flipped for local verification.